stories/Organism/Blocks/ContentCardWithAndWithoutImage/ContentCardWithAndWithoutImage.jsx

```diff
@@ -26,7 +26,7 @@ export const ContentCardWithAndWithoutImage = ({
                 </div>
                 <div className="content-card__caption">
                   <Heading type="5" label={item.contentname} />
-                  <Ctalink label={item.button} Type={item.btnType} />
+                  <span className="cta__link cta--space">{item.button}</span>
```

Comment:
Hey @neelesh-tippe  can we replace this mark with cta component eg :
`<Ctalink label={item.button} Type="Space" Emphasize="false"/>`

Reply:
Done